### ISSUE TITLE
Recent Blog description was displayed cut.

### ISSUE THREAD

**Issue by ermatitan** [titan-cebu/Rare-Parts#109]
![image](https://cloud.githubusercontent.com/assets/12791909/12226364/d21f596a-b855-11e5-81a7-c3621de04139.png)

**Comment by suniltitan** [comment]
As per the design 5 lines should be displayed.

**Comment by ermatitan** [comment]
Yes the design shows only 5 lines. But this should not be shorten since there is no link for the single page or a read more link. And also there is still a huge space below the arrow navigation, so why shorten the text description. Thanks
